kafka-connect-aws-s3/src/main/scala/io/lenses/streamreactor/connect/aws/s3/source/SourceContextReader.scala

```diff
@@ -39,7 +40,10 @@ object SourceContextReader {
       offsetMap <- Try(context().offsetStorageReader.offset(key).asScala).toOption.filterNot(_ == null)
       path      <- offsetMap.get("path").collect { case value: String => value }
```

Comment:
I suggest we create const values for these keys since avoids the same values in different files.